includes/admin/class.llms.admin.notices.php

```diff
@@ -252,6 +254,28 @@ public static function hide_notices() {
 		}
 	}
 
+	/**
+	 * Loads stored notice IDs from the database
+	 *
+	 * Handles potentially malformed data by ensuring that only an array of strings
+	 * can be loaded.
+	 *
+	 * @since [version]
+	 *
+	 * @return string[]
+	 */
+	protected static function load_notices() {
+
+		$option = get_option( 'llms_admin_notices', array() );
+
+		if ( ! is_array( $option ) ) {
+			$option = array( $option );
+		}
+
+		return array_filter( array_filter( $option, 'is_string' ) );
```

Comment:
```suggestion
		return array_filter( $option, function( $o ) {
			return ! empty( $o ) && is_string( $o );
		});
```
since closures are allowed now, we can avoid the double `array_filter()`, what do you think?